lib/scala/testkit/src/main/scala/org/enso/testkit/process/WrappedProcess.scala

```diff
@@ -0,0 +1,185 @@
+package org.enso.testkit.process
+
+import java.io._
+import java.util.concurrent.{Semaphore, TimeUnit}
+import scala.collection.Factory
+import scala.concurrent.TimeoutException
+import scala.jdk.CollectionConverters._
+import scala.jdk.StreamConverters._
+
+/** Represents a started and possibly running process. */
+class WrappedProcess(command: Seq[String], process: Process) {
```

Comment:
Note that this already existed in our test suite, it was only moved to the common sub-project.